goon/browserassets/css/browserOutput.css

```
@@ -304,8 +304,8 @@ h1.alert, h2.alert		{color: #000000;}
 .xenohighdanger			{color: #2a623d; 	font-weight: bold; font-size: 1.5em;}
 .xenoannounce           {color: #1a472a;    font-family: book-antiqua; font-weight: bold; font-style: italic; font-size: 1.5em;}
 .hivemind				{color: purple;		font-style: italic;}
-.xenoqueen				{					font-size: 1.6em;}
-.xenoshrike				{					font-size: 1.3em;}
+.xenoqueen				{					font-size: 18px;}
```

Why not just reduce it down by a percentage?
```suggestion
.xenoqueen				{					font-size: 1.4em;}
```

---

This is not a goal of this PR. Everything here uses PX and converting is for a larger job.